src/components/MenuBar/MenuBar.tsx

```diff
@@ -90,12 +90,12 @@ export default function MenuBar() {
               <ToggleVideoButton disabled={isReconnecting} />
               {!isSharingScreen && !isMobile && <ToggleScreenShareButton disabled={isReconnecting} />}
               {process.env.REACT_APP_DISABLE_TWILIO_CONVERSATIONS !== 'true' && <ToggleChatButton />}
+              <Menu />
             </Grid>
           </Grid>
           <Hidden smDown>
             <Grid style={{ flex: 1 }}>
-              <Grid container justifyContent="flex-end">
-                <Menu />
+              <Grid container justify="flex-end">
```

Comment:
This should be `justifyContent="flex-end"`. There's a warning in the console about it (but only when when browser width is reduced to show the mobile styles). 

![image](https://user-images.githubusercontent.com/12685223/128572250-a81d8d36-aaa0-478c-a728-9fc17a615b9c.png)